src/SubscribeMe/ValueObject/EmailAddress.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+declare(strict_types=1);
+
+namespace SubscribeMe\ValueObject;
+
+final class EmailAddress
+{
+    public function __construct(
+        private string $email,
+        private string $name
```

Comment:
Name should be optional 

```suggestion
        private ?string $name = null
```